### DISCUSSION TITLE
Debugging bundle size

### DISCUSSION THREAD

**Discussion by ATakaSKY** [vercel/next.js#47902] [Help]
### Summary

I've noticed an increase in the bundle size after removal of babelrc from the application. We're running on next 13.0.6.

Before babelrc removal: 
First load JS shared by all **159kb**

After babelrc removal: 
First load JS shared by all **174kb**

here's my babelrc:
```
{
  "presets": [
    [
      "next/babel",
      {
        "preset-env": {
          "useBuiltIns": false,
          "targets": "Chrome >= 60, Safari >= 10.1, iOS >= 10.3, Firefox >= 54, Edge >= 15"
        }
      }
    ]
  ],
  "plugins": [
    [
      "module-resolver",
      {
        "alias": {
          "server": "./src/server",
          "common": "./src/common",
          "types": "./src/types",
          "@my-icons": "./src/assets/icons",
          "@pack-atoms": "./node_modules/@package/dist/atoms",
          "@pack-components": "./node_modules/@package/dist/components",
          "@pack-config": "./node_modules/@package/dist/config",
          "@pack-helpers": "./node_modules/@package/dist/helpers",
          "@pack-molecules": "./node_modules/@package/dist/molecules",
          "@pack-typings": "./node_modules/@package/dist/typings"
        }
      }
    ],
    [
      "styled-components",
      {
        "ssr": true
      }
    ]
  ]
}

```

One thing I figured out here was the targets being set in babelrc, this was missing after removing babelrc. So I added them in package.json under browserlist and was able to brought it down by 8kbs. So now the first load JS shared by all is 166kb. 

What could be adding this another **7kbs(159 to 166)** is what is puzzling here. Any help is much appreciated.

**Comment by icyJoseph** [comment]
Use `@next/bundle-analyzer`, https://www.npmjs.com/package/@next/bundle-analyzer the latest version can even do a breakdown by entry point!

**Reply by icyJoseph** [reply]
Post back with results (to the extend that your project's  privacy policies allow), and such, happy to help you out figure out what's going on.
